01_load_exported_data_and_visualize_pupillometry.ipynb

```diff
@@ -13,15 +13,15 @@
     "\n",
     "Please check the documentation of [Pupil Player](https://docs.pupil-labs.com/#pupil-player) to see how to export data. \n",
     "This Notebook is accompanied by a [sample recording](https://drive.google.com/file/d/1Y3SkOLDiaYSVbDQ1VLMppyaIluoOPf06/view?usp=sharing) whose raw data was already exported. \n",
-    "To re-execute the cells below, please first download the recording and unpack it next to the notebook.\n",
+    "To re-execute the cells below, please first download the recording and unpack it inside the `recordings` directory next to the notebook.\n",
```

Comment:
Due to adding `recordings/` to `.gitignore`, there is no `recordings` folder next to the notebook when a user downloads/clones the repository for the first time. I see two possible solutions:
1. Add instructions to create the `recordings` folder
2. Add and commit a single (empty or README) file within the `recordings` folder, effectively adding the folder to the repository.

Reply:
good catch :+1: I added a `.gitkeep` to `recordings/` and excluded it from `.gitignore`